lib/netns_linux.c

```diff
@@ -262,6 +282,38 @@ static void ns_disable_internal(struct ns *ns)
 	}
 }
 
+/* VRF list existance check by name. */
+static struct ns_map_nsid *ns_map_nsid_lookup_by_nsid(ns_id_t ns_id)
+{
+	struct ns_map_nsid ns_map;
+
+	ns_map.ns_id = ns_id;
+	return (RB_FIND(ns_map_nsid_head, &ns_map_nsid_list, &ns_map));
+}
+
+ns_id_t ns_map_nsid_with_external(ns_id_t ns_id, bool maporunmap)
```

Comment:
maporunmap is not a good variable name.  I suggest just map

Reply:
ok